examples/verify_tsqrtP_calibration.py

```diff
@@ -0,0 +1,150 @@
+            "pulse_calib_5",
+        ),
+        (
+            "240819_amp0p1_calib_pulse_calib.h5",
+            "pulse_calib_2",
+        ),
+        (
+            "240819_amp0p2_calib_repeat_pulse_calib.h5",
+            "pulse_calib_9",
+        ),
+    ]:
+        d = psd.find_file(
+            filename, expno=nodename, exp_type="ODNP_NMR_comp/test_equipment"
+        )
+        assert (
+            d.get_prop("postproc_type") == "GDS_capture_v1"
+        ), "The wrong postproc_type was set so you most likely used the wrong script for acquisition"
+        amplitude = d.get_prop("acq_params")["amplitude"]
+        fl.basename = f"amplitude = {amplitude}"
+        if not d.get_units("t") == "s":
+            print(
+                "units weren't set for the t axis or else I can't read them from the hdf5 file!"
+            )
+            d.set_units("t", "s")
+        d *= V_atten_ratio
+        d /= np.sqrt(50)  # V/sqrt(R) = sqrt(P)
+        pulse_lengths = d.get_prop("programmed_t_pulse_us")
+
+        def switch_to_plot(d, j):
+            thislen = pulse_lengths[j]
+            fl.next(f"pulse length = {thislen}")
+
+        def indiv_plots(d, thislabel, thiscolor):
+            if skip_plots is None:
+                return
+            for j in range(len(d["beta"])):
+                if j % skip_plots == 0:
+                    switch_to_plot(d, j)
+                    fl.plot(
+                        d["beta", j],
+                        alpha=0.2,
+                        color=thiscolor,
+                        label="Amplitude = %f" % amplitude,
+                    )
+
+        # {{{ data is already analytic, and downsampled to below 24 MHz
+        indiv_plots(abs(d), "abs(analytic)", "orange")
+        d, nu_a, _ = find_apparent_anal_freq(d)
+        d.ft("t")
+        # {{{ Diagnostic to ensure the frequency was properly identified
+        fl.next("Frequency Domain")
+        fl.plot(d)
+        plt.text(
+            x=0.5,
+            y=0.5,
+            s=rf"$\nu_a={nu_a/1e6:0.2f}$ MHz",
+            transform=plt.gca().transAxes,
+        )
+        assert (0 > nu_a * 0.5 * HH_width) or (
+            0 < nu_a - 0.5 * HH_width
+        ), "unfortunately the region I want to filter includes DC -- this is probablye not good, and you should pick a different timescale for your scope so this doesn't happen"
+        # }}}
+        # {{{ apply frequency filter
+        d["t" : (None, nu_a - 0.5 * HH_width)] *= 0
+        d["t" : (nu_a + 0.5 * HH_width, None)] *= 0
+        # }}}
+        d.ift("t")
+        indiv_plots(abs(d), "filtered analytic", "red")
+        # {{{ Diagnostic to plot all pulses on the same plot to see none were skipped
+        fl.next("collect filtered analytic", legend=True)
+        for j in range(d.shape["beta"]):
+            s = d["beta", j].C
+            s["t"] -= abs(s).contiguous(lambda x: x > 0.03 * s.max())[0][0]
+            fl.plot(abs(s), alpha=0.3, label=f"{j}")
```

Comment:
There are a few problems here: 
- this is an unnecessary copy 
- you are repeating code that you do below 
- just doesn't make sense not to do this plot below, with the data that you actually use

Reply:
I've made this comment before!

Reply:
at this point thinking it over, now that we have the decreasing_ind thing this plot becomes obsolete and just extra noise so I am getting rid of it entirely

Reply:
you're not really responding to the comment here -- I'm just saying it's in the wrong place

Reply:
I just did it myself.